app/src/main/java/fr/free/nrw/commons/utils/ImageUtils.java

```diff
@@ -224,11 +228,23 @@ private static void setWallpaper(Context context, Bitmap bitmap) {
         try {
             wallpaperManager.setBitmap(bitmap);
             ViewUtil.showLongToast(context, context.getString(R.string.wallpaper_set_successfully));
+            if (progressDialog.isShowing() && progressDialog != null) {
```

Comment:
flip the conditions, null check should be done before

Reply:
@ashishkumar468 Sure!